src/main/java/at/hannibal2/skyhanni/data/MayorAPI.kt

```diff
@@ -0,0 +1,110 @@
+package at.hannibal2.skyhanni.data
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.config.ConfigManager
+import at.hannibal2.skyhanni.data.Mayors.Companion.setMayorWithActivePerks
+import at.hannibal2.skyhanni.data.jsonobjects.local.MayorJson
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.APIUtil
+import at.hannibal2.skyhanni.utils.CollectionUtils.put
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.SimpleTimeMark.Companion.asTimeMark
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
+
+object MayorAPI {
+    var lastUpdate = SimpleTimeMark.farPast()
+    private var dispatcher = Dispatchers.IO
+
+    private var rawMayorData: MayorJson? = null
+    var candidates = mapOf<Int, MayorJson.Candidate>()
+        private set
+    var currentMayor: Mayors? = null
+        private set
+    var timeTillNextMayor = Duration.ZERO
+        private set
+
+    private const val ELECTION_END_MONTH = 3
```

Comment:
Please add a comment for the value since 3 is not easy to understand that it is Late Spring. 
```suggestion
    private const val ELECTION_END_MONTH = 3 //Late Spring
```